druid/ingestion-spec/ml_observation_kafka.json

```diff
@@ -0,0 +1,475 @@
+            },
+            {
+              "type": "string",
+              "name": "childType"
+            },
+            {
+              "type": "string",
+              "name": "childExternalid"
+            },
+            {
+              "type": "string",
+              "name": "level"
+            },
+            {
+              "type": "string",
+              "name": "criteriaDescription"
+            },
+            {
+              "type": "string",
+              "name": "programDescription"
+            },
+            {
+              "type": "string",
+              "name": "solutionDescription"
+            },
+            {
+              "type": "string",
+              "name": "label"
+            },
+            {
+              "type": "string",
+              "name": "imp_project_id"
+            },
+            {
+              "type": "string",
+              "name": "imp_project_title"
+            },
+            {
+              "type": "string",
+              "name": "imp_project_goal"
+            },
+            {
+              "type": "string",
+              "name": "imp_project_externalId"
+            },
+            {
+              "type": "string",
+              "name": "ancestorName"
+            },
+            {
+              "type": "string",
+              "name": "scoringSystem"
+            },
+            {
+              "type": "string",
+              "name": "domainLevel"
+            },
+            {
+              "type": "string",
+              "name": "domainScore"
+            },
+            {
+              "name": "criteriaLevelReport",
+              "type": "boolean"
+            },
+            {
+              "type": "string",
+              "name": "user_schoolName"
+            },
+            {
+              "type": "string",
+              "name": "user_schoolId"
+            },
+            {
+              "type": "string",
+              "name": "user_schoolUDISE_code"
+            },
+            {
+              "type": "string",
+              "name": "solution_type"
+            },
+            {
+              "type": "string",
+              "name": "organisation_name"
+            },
+            {
+              "type": "string",
+              "name": "user_boardName"
+            }
+          ]
+        }
+      }
+    },
+    "metricsSpec": [
+      {
+        "type": "floatSum",
+        "name": "question_response_number",
+        "fieldName": "questionResponseLabel_number"
+      }
+    ],
+    "granularitySpec": {
+      "type": "uniform",
+      "segmentGranularity": "DAY",
+      "queryGranularity": "none",
+      "rollup": false
+    }
+  },
+  "tuningConfig": {
+    "type": "kafka",
+    "reportParseExceptions": false
+  },
+  "ioConfig": {
+    "topic": "<env>.ml.observation.druid",
+    "consumerProperties": {
+      "bootstrap.servers": "broker_ip:broker_port"
+    },
+    "taskCount": "<ml_observation_taskCount>",
```

Comment:
Same for most of the ingestion specs

Reply:
can you please share me the sample ingestion spec which is correct ? , because i go throughed the already existing ingestion spec of diksha.

Reply:
Resolved, Please check